sdks/golang/docs/RetryJobResponse.md

```
@@ -0,0 +1,93 @@
+# RetryJobResponse
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**JobId** | **int32** |  | 
```

markdownlint-cli2 (0.21.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)